open_api_framework/conf/base.py

```diff
@@ -670,3 +675,70 @@ def init_sentry(before_send: Callable | None = None):
 LOG_OUTGOING_REQUESTS_MAX_AGE = config(
     "LOG_OUTGOING_REQUESTS_MAX_AGE", default=7
 )  # number of days
+
+
+#
+# Django CSP settings
+#
+# explanation of directives: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+# and how to specify them: https://django-csp.readthedocs.io/en/latest/configuration.html
+#
+# NOTE: make sure values are a tuple or list, and to quote special values like 'self'
+
+# ideally we'd use BASE_URI but it'd have to be lazy or cause issues
+CSP_DEFAULT_SRC = [
+    "'self'",
+] + config("CSP_EXTRA_DEFAULT_SRC", default=[], split=True)
+
+# Allow any 'https:' host, as we don't know in advance which target is used by eHerkenning.
+# Behavior is also different between browsers regarding redirects, see:
+# https://stackoverflow.com/a/69439102 / https://github.com/w3c/webappsec-csp/issues/8
+CSP_FORM_ACTION = (
+    config(
+        "CSP_FORM_ACTION",
+        default=["\"'self'\"", "https:"]
+        + config("CSP_EXTRA_FORM_ACTION", default=[], split=True),
+        split=True,
+    )
+    + CORS_ALLOWED_ORIGINS
+)
+
+# * service.pdok.nl serves the tiles for the Leaflet maps (PNGs) and must be whitelisted
+# * the data: URIs are used by Leaflet (invisible pixel for memory management/image unloading)
+#   and the signature component which saves the image drawn on the canvas as data: URI
+CSP_IMG_SRC = (
+    CSP_DEFAULT_SRC
+    + ["data:", "https://service.pdok.nl/"]
```

Comment:
I think `service.pdok.nl` can be excluded (@annashamray double check, we don't use this anywhere in the registration components, right?)

Reply:
I think so, it's kadaster service afaik, which we don't use